src/workers/notifications/utils.ts

```diff
@@ -41,18 +41,43 @@ export async function articleNewCommentHandler(
   con: DataSource,
   commentId: string,
 ): Promise<NotificationHandlerReturn> {
-  const comment = await con
-    .getRepository(Comment)
-    .findOne({ where: { id: commentId }, relations: ['user'] });
+  const repo = con.getRepository(Comment);
+  const comment = await repo.findOne({
+    where: { id: commentId },
+    relations: ['user'],
+  });
   if (!comment) {
     return;
   }
   const postCtx = await buildPostContext(con, comment.postId);
   if (!postCtx) {
     return;
   }
+  const excludedUsers = [comment.userId];
+
+  const isReply = !!comment.parentId;
+  if (isReply && (postCtx.post.authorId || postCtx.post.scoutId)) {
+    const ids = [...new Set([postCtx.post.authorId, postCtx.post.scoutId])];
+    const threadFollower = await repo
+      .createQueryBuilder()
+      .select('"userId"')
+      .where({
```

Comment:
This one is not needed right?

Reply:
Oh, right. We can let this go. Thank you! 🙌